src/LRS.js

```diff
@@ -454,27 +455,35 @@ TinCan client library
             }
             if (typeof cfg.callback !== "undefined") {
                 callbackWrapper = function (err, xhr) {
-                    var result = xhr.responseText,
+                    var result,
```

Comment:
This didn't put the original back so is still broken, `result` needs to be initialized to `xhr` so that it carries through in the callback in the case of errors.